lib/ip.js

```diff
@@ -1226,6 +1226,9 @@ binet.isRoutable = function isRoutable(raw) {
  */
 
 binet.getNetwork = function getNetwork(raw) {
+  if (!binet.isRoutable())
```

Comment:
This needs `raw` to be passed in as the argument to `binet.isRoutable`

Reply:
The `bitcoind` implementation doesn't have the arguments, so I understand how this happened. See https://github.com/bitcoin/bitcoin/blob/47d981e8273804a040d71665a4cb16038d6717e1/src/netaddress.cpp#L315

Reply:
Ahh good catch, let me fix that right now.